builds/misc/templates/build-packages.yaml

```diff
@@ -232,6 +232,127 @@ stages:
+            inputs:
+              PathtoPublish: '$(build.artifactstagingdirectory)'
+              ArtifactName: 'iotedged-$(os)$(os_version)-$(arch)'
+            condition: succeededOrFailed()
+
+      ################################################################################
+      - job: mariner_linux_arm64
+      ################################################################################
+        displayName: Mariner_Linux on ARM64
+        pool:
+          name: $(pool.linux.arm.name)
+          demands:
+          - ImageOverride -equals agent-aziotedge-ubuntu-18.04-arm64
+        variables:
+          NugetSecurityAnalysisWarningLevel: warn
+        strategy:
+          matrix:
+            CBL-Mariner1.0-aarch64:
+              arch: aarch64
+              os: mariner
+              os_version: 1
+              mariner_release: 1.0-stable
+              target.iotedged: builds/mariner1/out/RPMS
+              target.logs: builds/mariner1/build/logs/pkggen/rpmbuilding/
+            CBL-Mariner2.0-aarch64:
+              arch: aarch64
+              os: mariner
+              os_version: 2
+              mariner_release: 2.0-stable
+              target.iotedged: builds/mariner2/out/RPMS
+              target.logs: builds/mariner2/build/logs/pkggen/rpmbuilding/
+
+        steps:
+          - bash: |
+              BASE_VERSION=`cat $BUILD_SOURCESDIRECTORY/edgelet/version.txt`
+              VERSION="$BASE_VERSION$BUILD_BUILDNUMBER"
+              echo "##vso[task.setvariable variable=VERSION;]$VERSION"
+              echo "##vso[task.setvariable variable=PACKAGE_ARCH;]$(arch)"
+              echo "PACKAGE_OS=$(os)"
+              echo "##vso[task.setvariable variable=MARINER_RELEASE;]$(mariner_release)"
+              echo "##vso[task.setvariable variable=OSVERSION;]$(os_version)"
+              mariner_arch=$(arch)
+              if [ $mariner_arch == "amd64" ]; then
+                mariner_arch="x86_64"
+              fi
+              echo "##vso[task.setvariable variable=MARINER_ARCH;]$mariner_arch"
+            displayName: Set Version
+
+          # remove this one 1ES can prevent the daiy update from triggering
+          - bash: |
+              set -ex
+              ps aux | grep -i apt
+              waitAttempts=60
+              sleepDuration=10
+              for (( i=0; $i<$waitAttempts; i++ ))
+              do
+                if [[ $(ps aux | grep -c apt.systemd.daily) == 1 ]]; then
+                  break
+                fi
+                sleep $sleepDuration
+              done
+              ps aux | grep -i apt
+            displayName: Wait for 1ES agent update
+
+          - bash: |
+              sudo apt install apt-transport-https ca-certificates curl software-properties-common -y
+              curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+              sudo add-apt-repository "deb [arch=arm64] https://download.docker.com/linux/ubuntu `lsb_release -cs` test"
```

Comment:
any reason why we are pulling it from the `test` ?

Reply:
not particularly will remove this.